### ISSUE TITLE
TestClient нужно закрывать безусловно

### ISSUE THREAD

**Issue by viskvortsov** [silverbulleters/vanessa-behavior#169]
Сейчас выпадает окно, которое запрашивает подтверждение.

**Comment by yuriybylinkin** [comment]
Так это окно тоже можно найти и нажать на кнопку "ОК" средствами
кнопконажималки?

8 декабря 2015 г., 11:16 пользователь viskvortsov <notifications@github.com>
написал:

> Сейчас выпадает окно, которое запрашивает подтверждение.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/silverbulleters/vanessa-behavior/issues/169>.
>



-- 
Былинкин Юрий

**Comment by viskvortsov** [comment]
Да, если делать интерактивно, но я сейчас говорю про функционал автозакрытия

**Comment by Pr-Mex** [comment]
Это зависит от конфигурации. Если идёт запрос подтверждения закрытия приложения, то его нельзя проигнорировать.
Или например может быть не сохраненный документ, тогда тоже будет появляться диалоговое окно.

**Comment by viskvortsov** [comment]
Мне кажется что в типовых должен быть похожий патерн. Может добавить шаг с условием, что если такая форма есть, то жать ОК

**Comment by Pr-Mex** [comment]
Предлагаю закрыть задачу.
И решать закрытие оставшихся сеансов 1С на CI.
